troposphere/serverless.py

```diff
@@ -45,7 +58,11 @@ class Function(AWSObject):
         'Policies': (policy_validator, False),
         'Environment': (Environment, False),
         'VpcConfig': (VPCConfig, False),
-        'Events': (dict, False)
+        'Events': (dict, False),
```

Comment:
The properties should be alphabetized (again, looks like they weren't to begin with).

Reply:
This is in the same order as defined [here](https://github.com/awslabs/serverless-application-model/blob/7067ff112ce762d97838e824ec97b9b25e453498/versions/2016-10-31.md). I do prefer this order though, this way you could follow SAM's example more easily. (So you don't have to rearrange them in your mind to get something to work)